### PR TITLE
Add fast-ai-detector raid-finetune submission

### PR DESCRIPTION
**Pull request by Ejhfast**
I'd like to score my model on test and potentially add to the leaderboard. Thanks!

### CODE REVIEW

**Comment by Ejhfast** [comment]
The evaluator on the previous submission couldn't find thresholds for FPR 5% and 1% for the "reviews" category, I think because the scores were quantized. Submitted non-quanitzed scores which hopefully should fix it

**Comment by Ejhfast** [comment]
Sorry for the repeated submissions here, I'm able to compute all metrics internally on held-out subsets so my best guess is the evaluator doesn't like logits for threshold search -- I now resubmitted as probability-scale scores.

**Comment by Ejhfast** [comment]
hey @liamdugan, can this be added to the leaderboard? thanks!

**Comment by liamdugan** [comment]
Sure thing @Ejhfast ! thanks for the submission
